test/src/org/labkey/test/tests/targetedms/TargetedMSUtilizationCalendarTest.java

```diff
@@ -0,0 +1,103 @@
+package org.labkey.test.tests.targetedms;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.components.targetedms.UtilizationCalendarWebPart;
+import org.labkey.test.pages.targetedms.PanoramaDashboard;
+import org.labkey.test.util.DataRegionTable;
+
+import java.util.Arrays;
+
+@Category({})
+@BaseWebDriverTest.ClassTimeout(minutes = 5)
+public class TargetedMSUtilizationCalendarTest extends TargetedMSTest
+{
+    @Override
+    protected @Nullable String getProjectName()
+    {
+        return getCurrentTestClass().getSimpleName() + " Project";
+    }
+
+    @BeforeClass
+    public static void initProject()
+    {
+        TargetedMSUtilizationCalendarTest init = (TargetedMSUtilizationCalendarTest) getCurrentTest();
+        init.doInit();
+    }
+
+    private void doInit()
+    {
+        setupFolder(TargetedMSTest.FolderType.QC);
+        importData(SProCoP_FILE);
+    }
+
+    @Test
+    public void testUtilizationCalendarActions()
```

Comment:
Please also check that the heat map is showing data from the imported Skyline file, and that toggling the heat map data source between samples and outliers works.

Reply:
Done